root/report/components/EventList.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {l} from '../../static/scripts/common/i18n';
+import PaginatedResults from '../../components/PaginatedResults';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import loopParity from '../../utility/loopParity';
+import formatDatePeriod
+  from '../../static/scripts/common/utility/formatDatePeriod';
+import {lp_attributes} from '../../static/scripts/common/i18n/attributes';
+import ArtistRoles from '../../static/scripts/common/components/ArtistRoles';
+import DescriptiveLink
+  from '../../static/scripts/common/components/DescriptiveLink';
+import type {ReportEventT} from '../types';
+
+const EventList = ({
+  items,
+  pager,
+}: {items: $ReadOnlyArray<ReportEventT>, pager: PagerT}) => (
+  <PaginatedResults pager={pager}>
+    <table className="tbl">
+      <thead>
+        <tr>
+          <th>{l('Event')}</th>
+          <th>{l('Type')}</th>
+          <th>{l('Artists')}</th>
+          <th>{l('Location')}</th>
+          <th>{l('Date')}</th>
+          <th>{l('Time')}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {items.map((item, index) => (
+          <tr className={loopParity(index)} key={item.event.id}>
+            <td>
+              <EntityLink
+                entity={item.event}
+                showDisambiguation
+                showEventDate={false}
+              />
+            </td>
+            <td>{item.event.typeName ? lp_attributes(item.event.typeName, 'item.event_type') : null}</td>
```

Comment:
Small context error:
```suggestion
            <td>{item.event.typeName ? lp_attributes(item.event.typeName, 'event_type') : null}</td>
```